code/game/objects/items/devices/radio/headset.dm

```diff
@@ -301,6 +301,8 @@
 	return
 
 /obj/item/device/radio/headset/AltClick(mob/user)
+	if(!istype(user) || !Adjacent(user) || user.incapacitated())

```

Comment:
who says it requires hands, we already have hands-free/voice activated shit in 2017 let alone 2557